mcs/class/System/System.Security.Cryptography.X509Certificates/X509Helper2.cs

```diff
@@ -122,35 +122,35 @@ internal static void ExportAsPEM (MonoBtlsX509 x509, Stream stream, bool include
 				x509.ExportAsPEM (bio, includeHumanReadableForm);
 			}
 		}
-#endif // !FEATURE_NO_BSD_SOCKETS
+#endif // !MONO_FEATURE_BTLS
```

Comment:
@baulig it seems the FEATURE_NO_BSD_SOCKETS was used as a shortcut for disabling this on WatchOS, with my changes it'll be disabled on all monotouch platforms (which should be OK since accessing BTLS wouldn't have worked on monotouch).

Can you please double-check this is ok? It seems to compile fine at least.